x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -123,33 +126,34 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
     mutedInstanceIds: {
```

Comment:
I'm surprised we need to keep this indexed!

Reply:
Yes there is one API integration test who is using it